apps/io-session-manager-internal/src/controllers/__tests__/auth-lock.test.ts

```diff
@@ -0,0 +1,146 @@
+import { conflictError, toGenericError } from "../../utils/errors";
+
+const aFiscalCode = "SPNDNL80R13C555X";
+
+const mockedDependencies = {
+  SessionService: SessionServiceMock,
+  // service is already mocked, no need to mock the repositories
+  RedisRepository: {} as RedisRepository,
+  SafeRedisClientTask: TE.of({} as RedisClusterType),
+  FastRedisClientTask: TE.of({} as RedisClusterType),
+  AuthLockRepository: {} as AuthLockRepository,
+  AuthenticationLockTableClient: mockTableClient,
+  LollipopRepository: {} as LollipopRepository,
+  RevokeAssertionRefQueueClient: mockQueueClient,
+  InstallationRepository: {} as InstallationRepository,
+  NotificationQueueClient: mockQueueClient,
+};
+describe("Auth Lock Handler", () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  const aValidUnlockCode = "000000000" as UnlockCode;
+  const aValidPayload = {
+    unlock_code: aValidUnlockCode,
+  };
+
+  it("should succeed locking the authentication of a user", async () => {
+    const req = {
+      ...H.request("mockUrl"),
+      path: {
+        fiscalCode: aFiscalCode,
+      },
+      body: aValidPayload,
+    };
+    const result = await makeAuthLockHandler({
+      ...httpHandlerInputMocks,
+      input: req,
+      ...mockedDependencies,
+    })();
+
+    expect(result).toMatchObject(E.right(H.empty));
+  });
+
+  it.each`
+    scenario     | value
+    ${"missing"} | ${undefined}
+    ${"empty"}   | ${null}
+    ${"invalid"} | ${{ unlock_code: "abc" }}
+  `("should return Bad request on $scenario body", async ({ value }) => {
+    const req = {
+      ...H.request("mockUrl"),
+      path: {
+        fiscalCode: aFiscalCode,
+      },
+      body: value,
+    };
+
+    const result = await makeAuthLockHandler({
+      ...httpHandlerInputMocks,
+      input: req,
+      ...mockedDependencies,
+    })();
+
+    expect(result).toMatchObject(E.right({ body: { status: 400 } }));
+  });
+
+  it("should return Bad request on invalid path param", async () => {
+    const req = {
+      ...H.request("mockUrl"),
+      path: {
+        fiscalCode: "abc",
+      },
+      body: aValidPayload,
+    };
+
+    const result = await makeAuthLockHandler({
+      ...httpHandlerInputMocks,
+      input: req,
+      ...mockedDependencies,
+    })();
+
+    expect(result).toMatchObject(E.right({ body: { status: 400 } }));
+  });
+
+  it("should return Error internal coherent with service response", async () => {
+    const req = {
+      ...H.request("mockUrl"),
+      path: {
+        fiscalCode: aFiscalCode,
+      },
+      body: aValidPayload,
+    };
+    mockLockUserAuthentication.mockReturnValueOnce(
+      RTE.left(toGenericError("ERROR")),
+    );
+
+    const result = await makeAuthLockHandler({
+      ...httpHandlerInputMocks,
+      input: req,
+      ...mockedDependencies,
+    })();
+
+    expect(mockLockUserAuthentication).toHaveBeenCalledTimes(1);
+    expect(result).toMatchObject(E.right({ body: { status: 500 } }));
```

Comment:
```suggestion
      E.right({ body: { status: 500, title: "ERROR" } }),
```

Reply:
I included titles in those tests in https://github.com/pagopa/io-auth-n-identity-domain/pull/295/commits/1e92e233d29ebccefb945165c788cdd087e5b18f